src/ansys/optislang/core/project.py

```diff
@@ -228,6 +228,60 @@ def get_status(self) -> str:
         """
         return self.__osl_server.get_project_status()
 
+    def _get_project_tree(self) -> list:
+        """Return the project tree in a list format.
+
+        Returns
+        -------
+        List
+            List with all the nodes in the project tree.
+        """
+        full_project_tree = self.__osl_server.get_full_project_tree()
+        project_tree = [
+            {
+                "uid": full_project_tree["projects"][0]["system"]["uid"],
+                "name": full_project_tree["projects"][0]["system"]["name"],
+                "is_root": True,
+                "kind": full_project_tree["projects"][0]["system"]["kind"],
+                "level": 0,
+            }
+        ]
+        return self._get_child_nodes(
+            full_project_tree["projects"][0]["system"]["nodes"], project_tree
+        )
+
+    def _get_child_nodes(self, node_properties: dict, project_tree: list) -> list:
+        """Recursively walk throughout the full_project_tree and collect the nodes.
```

Comment:
```suggestion
        """Recursively walk throughout the full project tree and collect the nodes.
```
Avoid using "code" in descritions. Explain in simple English.